### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by Alejandro-Bernal-M**
Implementation of work section constructed dynamically and popup windows

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -269,6 +269,103 @@ li {
...
+}
+
+.popup-show {
+  z-index: 100;
+  opacity: 1;
+  background: #c1c7d0;
+  width: 100%;
+  height: auto;
+  position: absolute;
+}
```
Comment (brainconnect93) [inline styles.css]: - [ ] Hi @Alejandro-Bernal-M, great effort working on the pop-up modal section of your project, 👍 👍  but currently the blur effect given to the main page when the pop-up card is displayed is not implemented and this makes your work looks different from what we have in the Figma design. you can simply make use of the RGBA color format for the background-color or target the container holding the pop-up cards and give it a `backdrop-filter: blur(10px)` CSS property.

Your Design 👇 

alex


>N.B: Please kindly implement this changes for both `mobile` and `desktop` version

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @Alejandro-Bernal-M, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (Alejandro-Bernal-M) [reply styles.css]: Thank you for the correction. I appreciate your recommendation and your effort.

**Review by mwanawabangona — APPROVED** [review]
Hi @Alejandro-Bernal-M  ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**